### PR TITLE
Update version-runtime to 2.1.5 and date to correct release date

### PR DESCRIPTION
**Pull request by Rutix**
I noticed when updating our builds that the version-runtime and date was wrong at these two places.

### CODE REVIEW

**Comment by Rutix** [comment]
@karelz Any chance this can be merged :)?

**Comment by karelz** [comment]
I let @leecow @vivmishra to sign off. I don't know how and where the files are used and if it is ok to just change them

File: release-notes/releases.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "version-runtime-display": "2.1.5",
         "version-sdk-display": "2.1.403",
         "version-aspnetcoremodule": "8.2.1991.0",
-        "date": "2018-09-11",
+        "date": "2018-02-10",
```
Comment (nil4) [inline release-notes/releases.json]: Did you mean `2018-10-02` (i.e. Oct 2nd, not Feb 10th)

Comment (Rutix) [reply release-notes/releases.json]: Oops forgot that US switches the day and month. Fixed it in the next commit :)

**Comment by vivmishra** [comment]
Thanks for reporting.

**Pull request merged**